### DISCUSSION TITLE
Can I compute the velocity or orbital magnetic moment matrix using wannierberri?

### DISCUSSION THREAD

**Discussion by sharmashivamm16** [wannier-berri/wannier-berri#303] [General]
I have been able to calculate the tabulated value of velocity using, which is a sum over all Wannier bands but I want to calculate the matrix $v_{nn'}$ and also the matrix for other tabulators, is there a way to do that?

**Comment by stepan-tsirkin** [comment]
@sharmashivamm16 , there is an old pull request, for tabulating off-diagonal matrix elements #184
It was not merged then due to lack of feedback. Could you try this and comment there what you think of it.
